controller/iteration_blackbox_test.go

```diff
@@ -1080,3 +1080,193 @@ func (rest *TestIterationREST) TestIterationDelete() {
 		}
 	})
 }
+
+func (rest *TestIterationREST) TestUpdateIteration() {
+	resetFn := rest.DisableGormCallbacks()
+	defer resetFn()
+	rest.T().Run("update success - iteration parent", func(t *testing.T) {
+		// build following structure
+		// 	root 0
+		// 		|---- itr 1
+		//			|---- itr 2
+		//				|---- itr 3
+		//  				|---- itr 4
+		// 			|---- itr 5
+		// 		|---- itr 6
+
+		// given
+		fxt := tf.NewTestFixture(t, rest.DB,
+			tf.Iterations(7, tf.SetIterationNames("root iteration", "iteration 1",
+				"iteration 2", "iteration 3", "iteration 4", "iteration 5", "iteration 6"),
+				func(fxt *tf.TestFixture, idx int) error {
+					itr := fxt.Iterations[idx]
+					switch idx {
+					case 1:
+						itr.MakeChildOf(*fxt.Iterations[0])
```

Comment:
I suggest to use `fxt.IterationByName` instead of accessing the array directly. It's easier to read.